components/IscnRegisterForm.vue

```diff
@@ -874,6 +877,16 @@ export default class IscnRegisterForm extends Vue {
   }
 
   async calculateISCNFee(): Promise<void> {
+    try {
+      const [ data ] = await Promise.all(
+        this.likerIds.map((e) =>
+          this.$axios.get(getLikerIdMinApi(e as string),
```

Comment:
I think we should add .catch for each .get to prevent 1 invalid entry throwing all promises